helix-view/src/clipboard.rs

```diff
@@ -88,6 +88,11 @@ pub fn get_clipboard_provider() -> Box<dyn ClipboardProvider> {
             primary_paste => "wl-paste", "-p", "--no-newline";
             primary_copy => "wl-copy", "-p", "--type", "text/plain";
         }
+    } else if exists("win32yank.exe") {
+        command_provider! {
+            paste => "win32yank.exe", "-o", "--lf";
+            copy => "win32yank.exe", "-i", "--crlf";
+        }
```

Comment:
Can you place it in the same position it was previously, after `xsel`? That way it's not in the middle of `Wayland`/`X11`.

Reply:
Done!